eslint.config.mjs

```diff
@@ -0,0 +1,15 @@
+// @ts-check
+
+import eslint from '@eslint/js';
+import tseslint from 'typescript-eslint';
+
+export default tseslint.config(
+  eslint.configs.recommended,
+  ...tseslint.configs.recommended,
```

Comment:
As I mentioned in standup, there's a config for linting with type information, but I didn't include that for now because we aren't using it in the SDK: https://typescript-eslint.io/getting-started/typed-linting

It looks like client-logger does use it, though: https://github.com/purecloudlabs/genesys-cloud-client-logger/blob/develop/.eslintrc.js#L20